desktop/app/kbfs-helper.js

```diff
@@ -1,10 +1,32 @@
 // @flow
 
 import {ipcMain, shell} from 'electron'
+import fs from 'fs'
+import {pathToURL} from './paths'
 
 export default function () {
   ipcMain.on('openInKBFS', (e, path) => {
-    // We do the `shell.openItem` here to not block the ui thread
-    shell.openItem(path)
+    fs.stat(path, (err, stats) => {
```

Comment:
Yes it is. I've profiled the app and am finding a pretty large hand there though. Unfortunately, the place it says it's hanging isn't making any sense, so I'm trying to debug more with some console.log/performance.now stuff, but I'm running into some weird issues of not being able to debug the code in devtools.